runtimes/pythonrt/Makefile

```diff
@@ -0,0 +1,44 @@
+ifeq ($(shell uname),Linux)
+	open=xdg-open
+else
+	open=open
+endif
+
+all:
+	$(error please pick a target)
```

Comment:
i would just make it run ci

Reply:
It's an old habit of mine to require explicit targets. Not that important to me, will change